Makefile

```diff
@@ -1,7 +1,6 @@
 .PHONY: generate
 
 generate:
-	docker run --rm -v  ${CURDIR}:/local openapitools/openapi-generator-cli generate -i /local/petstore-openapi.yaml -g go-server --additional-properties=addResponseHeaders=true,router=chi -o /local/api
-	# The generated demo service and module configuration confuse the compiler, and we're not using them, so remove them
-	del api\main.go api\go.mod
+	docker run --rm -v  ${CURDIR}:/local openapitools/openapi-generator-cli generate -i /local/petstore-openapi.yaml -g go-server --additional-properties=addResponseHeaders=true,outputAsLibrary=true,onlyInterfaces=true,router=chi -o /local/api
+
```

Comment:
Remove the provided service boilerplate in future generations, and provide the service stubs as a package instead